database/insertData.js

```diff
@@ -252,11 +259,9 @@ const queries = [
                       category: QUESTION
                       parameters: {
                         label: "Select Importer"
-                        options: [
-                          "Importer A",
-                          "Importer B",
-                          "Importer C"
-                        ]
+                        placeholder: "Select"
+                        options: ["Importer A", "Importer B", "Importer C"]
```

Comment:
```suggestion
                        options: [
                          "Importer A", 
                          "Importer B", 
                          "Importer C"
                        ]
```

I think that's easier to read

Reply:
Need to be careful here as the Query editor doesn't want commas when listing vertically, so it's non-standard JSON.

Also -- I don't think we want to get in the habit of over-riding the auto-formatting. I always copy this whole query into the Web GUI to check it's correct, and it auto-formats it -- and we don't want to have to remember to do a bunch of manual changes every time we paste it back.